taps/taps_test.go

```diff
@@ -81,23 +81,23 @@ func TestCreateRemoveTaps(t *testing.T) {
 	}
 }
 
-func TestCreateRemoveExtra(t *testing.T) {
```

Comment:
No, you need to use [Skip()](https://golang.org/pkg/testing/#hdr-Skipping). Also, add a comment that we disabled this test as it breaks in GitHub Actions and it doesn't seem essential for the test coverage

Reply:
done